interface.html

```diff
@@ -54,93 +53,13 @@
             </div>
           </div>
         </div>
+      </div>
 
-        <!-- <div class="title-header">
-          <p>Thumbnail options</p>
-          <p>You can enable a thumbnail on your main list and/or detail views. Configure below where and how to show the thumbnails.</p>
+      <div class="options-no-columns">
+        <div class="alert alert-warning">
+          Data source appears to be empty or misconfigured. Please visit <a href="/data-sources" target="_blank">Manage app data</a> to make sure the data source has data entries.
```

Comment:
This link goes to the API (probably because of the iframe URL), it should go to studio.